### DISCUSSION TITLE
About  GitHub Foundations Microsoft Learn Modules

### DISCUSSION THREAD

**Discussion by Sakib99-cyber** [community/community#142923] [GitHub Education]
### Hi everyone,

GitHub Foundations Microsoft Learn Modules, as a GitHub student,
if i complete any course from there,
will i get free certificate without any payment or card?

**Comment by davevad93** [comment]
Hi @Sakib99-cyber ,

The [GitHub Foundations training path](https://learn.microsoft.com/en-us/training/paths/github-foundations/) is a preparation for the certification exam for the GitHub Foundations certificate which is now included in the `Student Developer Pack` for free and considering you are a verified student, you can take the exam and get the certificate for free. Although there has been a little bug and the offer is temporarily disabled, however it's expected to resume by October 31st. For more information and updates, subscribe to this discussion [#138834.](https://github.com/orgs/community/discussions/138834)

**Comment by Jacksx20** [comment]
May I ask if it has been restored at present
